tasks/copy_to_archive/test/unit_tests/test_copy_to_archive.py

```diff
@@ -147,7 +146,7 @@ def test_handler_happy_path(self, mock_task: MagicMock):
                 "collectionVersion": uuid.uuid4().__str__(),
             },
         }
-        handler_input_context = LambdaContextMock()
```

Comment:
LambdaContextMock can be removed, unless I'm missing something.

Reply:
yes already changed it to
 handler_input_context = Mock()

Reply:
I'm referring to the class itself. Unless I'm missing something, our code should have to references to "LambdaContextMock" in any file.

Reply:
you mean `no` references?

Reply:
Correct

Reply:
removed helpers.py